### PR TITLE
Ldanzinger/cut geometry

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review/merge

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+  // create graphics for the U.S. side
+  Graphic* usSide = new Graphic(this);
+  usSide->setGeometry(geoms.at(0));
+  SimpleLineSymbol* outline = new SimpleLineSymbol(SimpleLineSymbolStyle::Solid, QColor("blue"), 1.0f, this);
+  usSide->setSymbol(new SimpleFillSymbol(SimpleFillSymbolStyle::ForwardDiagonal, QColor("#40ff00"), outline, this));
+
+  // create graphics for the Canada side
+  Graphic* canadaSide = new Graphic(this);
+  canadaSide->setGeometry(geoms.at(1));
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp]: Can `geoms.at(1)` just be passed into the constructor instead?

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+// Cuts the Lake Superior polygon into two and symbolizes each output graphic
+void CutGeometry::cutPolygon()
+{
+  // perform the cut
+  QList<Geometry> geoms = GeometryEngine::cut(m_lakeSuperiorGraphic->geometry(), m_borderGraphic->geometry());
+
+  // create graphics for the U.S. side
+  Graphic* usSide = new Graphic(this);
+  usSide->setGeometry(geoms.at(0));
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp]: Can `geoms.at(0)` just be passed into the constructor instead?

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+// Create the two baseline graphics
+void CutGeometry::createGraphics()
+{
+  // Lake Superior Graphic
+  m_lakeSuperiorGraphic = new Graphic(this);
+  SimpleLineSymbol* outline = new SimpleLineSymbol(SimpleLineSymbolStyle::Solid, QColor("blue"), 1.0f, this);
+  m_lakeSuperiorGraphic->setSymbol(new SimpleFillSymbol(SimpleFillSymbolStyle::Solid, QColor(198, 204, 255, 178), outline, this));
+  m_lakeSuperiorGraphic->setGeometry(createLakeSuperiorPolygon());
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp]: Same here, use the Graphic constructor instead for setting the symbol and graphic.

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  m_overlay->graphics()->append(m_lakeSuperiorGraphic);
+
+  // Border Graphic
+  m_borderGraphic = new Graphic(this);
+  m_borderGraphic->setSymbol(new SimpleLineSymbol(SimpleLineSymbolStyle::Dot, QColor("red"), 5.0f, this));
+  m_borderGraphic->setZIndex(10);
+  m_borderGraphic->setGeometry(createBorderPolyline());
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp]: Use Graphic constructor for symbol and geometry

**Comment by ldanzinger** [comment]
@michael-tims thanks, updated

**Pull request merged**